### PR TITLE
Customer GraphQL Uid changes

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
## Problem

Customer GraphQL Schema with UID changes.

## Solution

To keep consistency throughout the GraphQL schema, we have replaced the Id field with Uid.

### CODE REVIEW

**Comment by RakeshJesadiya** [comment]
@melnikovi @cpartica Please validate the schema file and let me know if need any improvement.

File: design-documents/graph-ql/coverage/customer/customer.graphqls [hunk 1]
```diff
@@ -100,17 +100,17 @@ type Customer @doc(description: "Customer defines the customer name and address
...
     is_subscribed: Boolean @doc(description: "Indicates whether the customer is subscribed to the company's newsletter") @resolver(class: "\\Magento\\CustomerGraphQl\\Model\\Resolver\\IsSubscribed")
     addresses: [CustomerAddress] @doc(description: "An array containing the customer's shipping and billing addresses") @resolver(class: "\\Magento\\CustomerGraphQl\\Model\\Resolver\\CustomerAddresses")
     gender: Int @doc(description: "The customer's gender (Male - 1, Female - 2)")
 }
 
 type CustomerAddress @doc(description: "CustomerAddress contains detailed information about a customer's billing and shipping addresses"){
-    id: Int @doc(description: "The ID assigned to the address object")
-    customer_id: Int @doc(description: "The customer ID") @deprecated(reason: "customer_id is not needed as part of CustomerAddress, address ID (id) is unique identifier for the addresses.")
+    uid: ID @doc(description: "The ID assigned to the address object")
+    customer_iid: ID @doc(description: "The customer ID") @deprecated(reason: "customer_id is not needed as part of CustomerAddress, address ID (id) is unique identifier for the addresses.")
```
Comment (dthampy) [inline design-documents/graph-ql/coverage/customer/customer.graphqls:111]: please fix the typo.

Comment (RakeshJesadiya) [reply design-documents/graph-ql/coverage/customer/customer.graphqls]: I have made changes for the type. Please check it.

**Comment by RakeshJesadiya** [comment]
@dthampy Please check updated PR.

**Review by cpartica — APPROVED** [review]

**Pull request merged**